hardware/MQTT.cpp

```diff
@@ -3126,32 +3176,20 @@ void MQTT::InsertUpdateSwitch(_tMQTTASensor* pSensor)
 			{
 				szOnOffValue = "Set Level";
 				// Make level relative to 100.
-				if (
-					(sSwitchType == STYPE_BlindsInverted)
-					|| (sSwitchType == STYPE_BlindsPercentageInverted)
-					|| (sSwitchType == STYPE_BlindsPercentageInvertedWithStop)
-					)
-				{
+				if (sSwitchType == STYPE_BlindsInverted || sSwitchType == STYPE_BlindsPercentageInverted || sSwitchType == STYPE_BlindsPercentageInvertedWithStop)
 					// invert level for inverted blinds with percentage.
 					level = (int)((100.0 / (pSensor->position_open - pSensor->position_closed)) * level);
-				}
 				else
-				{
 					level = (int)(100 - ((100.0 / (pSensor->position_open - pSensor->position_closed)) * level));
-				}
 			}
 		}
 		if (!root["color"].empty())
 		{
 			Json::Value root_color;
 			bool ret = ParseJSon(sColor, root_color);
 			if (ret)
-			{
```

Comment:
Could you undo the formatting here. It makes the code harder to read

Reply:
I do not quite agree to the readability here, but will do so.